src/main/java/uk/gov/ida/verifyserviceprovider/services/ResponseService.java

```diff
@@ -72,16 +70,30 @@ public TranslatedResponseBody convertTranslatedResponseBody(
                 return responderCodeTranslator.translateResponderCode(statusCode);
             case StatusCode.SUCCESS:
                 List<Assertion> assertions = assertionDecrypter.decryptAssertions(validatedResponse);
+                if (assertionsContainEidasUnsignedAssertionsResponse(assertions)) {
+                    UnsignedAssertionsResponseHandler handler = unsignedAssertionsResponseHandler.get();
+
+                    ValidatedResponse validatedCountryResponse = handler.getValidatedResponse(assertions.get(0), expectedInResponseTo);
```

Comment:
_Personal style choice (again!)..._ when putting a constant like `0` into code, it feels awkward not to explain the assumption behind it (ie. "in this case there can only be 1 assertion and it should contain...") I appreciate we don't write many comments though - so I suspect it's ok not to do this.

Reply:
I like that, I'm going to do it.